modules/ocf_lb/templates/keepalived.conf.erb

```diff
@@ -6,7 +6,7 @@ vrrp_instance VI_1 {
 
     authentication {
         auth_type PASS
-        auth_pass $ <%= @keepalived_secret %>
+        auth_pass $ <%= @secret %>
```

Comment:
@keur why is there a `$` here?

Reply:
I would double check and see if it is even getting rendered on the hosts. I skimmed the manpage and it seems okay to remove. This was largely ported from the `ocf_mesos` code, so this is a blindly copied from there. And god that diff was fucking hard to find. This is why there was literally no point at all for us to remove the ocf_mesos modules rather than just remove them from hiera. There is lots of context that influences today's design in the old code. Sigh.

https://github.com/ocf/puppet/blob/8b6cb903d287f7544282bfc78355283dce2a0eb6/modules/ocf_mesos/templates/master/load_balancer/keepalived.conf.erb#L9